api/package.json

```diff
@@ -98,6 +97,7 @@
         "serverless-prune-plugin": "^2.1.0",
         "ts-jest": "^29.1.1",
         "ts-loader": "^9.4.4",
+        "tsx": "^4.19.2",
```

Comment:
Can ts-node be uninstalled now?

Reply:
That's the weird thing; I couldn't see that it was installed in the first place!